miscellaneous-commands/userconfig.md

```diff
@@ -22,8 +22,9 @@ For more information on editing permission requirements for specific users/roles
 /userconfig settings [reaction-role-notifications] [vote-reminders]
 ```
 
-* `reaction-role-notifications`: <TODO: describe>
-* `vote-reminders`: <TODO: describe>
+* `reaction-role-notifications`: This subcommand lets you enable or disable notifications when you add or remove [a reaction role](/start-up/setting-up-reaction-roles.md) from yourself.
```

Comment:
option, not subcommands

this one if set to false disables them regardless of server settings on the matter, if true, the server decides whether or not to send the notification

default is true